### ISSUE TITLE
EDDI not letting me customize, nor letting me use my customized personality.

### ISSUE THREAD

**Issue by Luckfish3** [EDCD/EDDI#290]
## EDDI version in which issue found
[]2.4.6-b1

## Steps to reproduce

1. Started EDDI, now it always starts with the 'default' personality instead of the one I created. This has only started happening recently. Also, I've tried modifying the script for 'message recieved' to cut out a lot of the neverending NPC chatter, and it doesn't change. I've also un-ticked the 'message recieved' box, and EDDI still reads out every NPC message. 
2. [the more reproducible, the better our chances of fixing it]

## Expected
[describe what you expected to happen] 
EDDI to start up with what profile I choose, and to let me modify the responses. 

## Observed
[describe what happened instead]
EDDI starts with the default profile, and even after I select my customized one it still acts like the default one. 

## Investigation
[any investigation you have done, regression tests against earlier versions, etc]

**Comment by Tkael** [comment]
Sounds like EDDI may be having trouble reading or writing to your custom settings at %APPDATA%/EDDI. Could your folder permissions have recently changed?

Are you using EDDI by itself or with VoiceAttack? If VoiceAttack is running while you are using EDDI, did you start it by using the `Configure EDDI` command or did you open a separate instance?

Can you enable verbose logging, switch to your custom personality, make a change, switch back to to the default EDDI personality, and then hit the `Report an Issue` button. A zipped log file will appear on your desktop - attach it here then disable verbose logging again.

**Comment by Luckfish3** [comment]
Not using VA at all, I've got plenty of buttons on my HOTAS, and EDDI does all the talking I need. Regarding folder permissions, this is a brand-new computer, running Win10 64-bit (same as old computer). Perhaps Windows has some default setting preventing writing to my custom settings?

ZIP file attached. I modified the 'Message Received' script slightly.
[eddi_issue.zip](https://github.com/EDCD/EDDI/files/1540716/eddi_issue.zip)

**Comment by Tkael** [comment]
No, default settings should be fine. I'm actually using Windows 10 64 bit myself.

Ok. Per your log it is definitely a problem with EDDI not being able to find and read your personality file:
```
2017-12-07T21:41:04 Files:Read [E] FileC:\Users\luckf\AppData\Roaming\EDDI\personalities\jess.json not found System.IO.FileNotFoundException: Could not find file 'C:\Users\luckf\AppData\Roaming\EDDI\personalities\jess.json'.
2017-12-07T21:41:04 SpeechResponder:Reload [W] Failed to find named personality; falling back to default 
```
Please confirm that jess.json exists at the location EDDI is searching? If the file path is correct, this is likely still a permissions / file ownership issue. Did you download this personality from a [website](http://snafets.de/elite/eddi_scripts/index.php)? If so, try the steps described [here](http://www.thewindowsclub.com/take-ownership-windows-8) to attempt to take ownership of the file?

**Comment by Luckfish3** [comment]
I had downloaded it from the site you linked to...I had uploaded it in the past, and figured that downloading it would be an easy way to get the personality on my new computer. 

I went to the Windows Club page you linked to, and followed the instructions to take ownership of the C:\Users\luckf file and everything inside of it. Unfortunately, EDDI still kept defaulting  to the default profile after closing and re-opening it. On a whim, I (within EDDI) made a copy of the personality, and left it selected when I closed EDDI. Upon re-opening EDDI, it kept the copy (Jess 2) selected. If I swapped back to the original one (Jess) and closed EDDI, it would jump back to the default one. 

Perhaps EDDI just didn't like the one I downloaded, even after changing permissions. Either way, EDDI is working well on the personality copy I made, so I'll just probably delete the Jess one and stick with the copy. Thank you for your help, EDDI is by far my favorite 3rd party app for this game!
